fashrevwall/models/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+class Tweet(models.Model):
+    text = models.CharField(max_length=200)
+    image_url = models.CharField(max_length=200)
+    user = models.CharField(max_length=100)
```

Comment:
At the end we should have a return to get the image url as that is what we care most about and it identifies the tweet in this case:
`` `	def __unicode__ (self):
		return self.image_url